obs_maven/repo.py

```diff
@@ -58,6 +60,18 @@ def find_primary(self):
 
     def parse_primary(self):
         primary_url = self.find_primary()
+
+        try:
+            # Check if we have this primary file cached
+            cache_file = os.path.join(self.cache_dir, primary_url.rsplit('/', 1)[1] + '.data')
+            if os.path.exists(cache_file):
+                logging.debug("Loading RPMs from cache file: %s", cache_file)
+                with open(cache_file, 'rb') as fd:
+                    self._rpms = pickle.load(fd)
+                    return
+        except OSError as error:
+            logging.warn("Error loading RPMs from cache: %s", error)
```

Comment:
```suggestion
            logging.warning("Error loading RPMs from cache: %s", error)
```
`logging.warn` is deprecated.

Reply:
fixed